File: gdx/src/com/badlogic/gdx/maps/tiled/TiledMapRenderer.java
package com.badlogic.gdx.maps.tiled;

import com.badlogic.gdx.maps.MapLayer;
import com.badlogic.gdx.maps.MapObject;
import com.badlogic.gdx.maps.MapRenderer;

public interface TiledMapRenderer extends MapRenderer {
    void renderObjects(MapLayer layer);

    void renderObject(MapObject object);

    void renderTileLayer(TiledMapTileLayer layer);

    void renderImageLayer(TiledMapImageLayer layer);
}

File: gdx/src/com/badlogic/gdx/maps/tiled/TiledMapRenderer.kt
package com.badlogic.gdx.maps.tiled

import com.badlogic.gdx.maps.MapLayer
import com.badlogic.gdx.maps.MapObject
import com.badlogic.gdx.maps.MapRenderer

/**
 * Info : This interface was moved to Kerman game engine.
 */
interface TiledMapRenderer : MapRenderer {
    fun renderObjects(layer: MapLayer)
    fun renderObject(mapObject: MapObject)
    fun renderTileLayer(layer: TiledMapTileLayer)
    fun renderImageLayer(layer: TiledMapImageLayer)
}
